vispy/plot/plot.py

```diff
@@ -31,15 +35,19 @@ def image(*args, **kwds):
     """
     canvas = SceneCanvas(keys='interactive')
     canvas.view = canvas.central_widget.add_view()
-    canvas.image = visuals.Image(*args, **kwds)
+    canvas.image = visuals.Image(parent=canvas.view.scene, *args, **kwds)
     canvas.view.add(canvas.image)
     canvas.show()
-    canvas.view.camera.invert_y = False
-    canvas.view.camera.auto_zoom(canvas.image)
+    canvas.view.camera = cameras.PanZoomCamera(fixed_ratio=True,
+                                               scale_ratio=(1, -1, 1),
+                                               )
     plots.append(canvas)
+    # todo: (AK) I think this should return an image
+    # Also, wtf: this creates an image in a viewbox in a widget in a canvas.
```

Comment:
What's the problem with that? Each object has its own purpose; that's separation of concerns.